cluster-autoscaler/cloudprovider/gce/templates.go

```diff
@@ -598,6 +598,8 @@ func ToSystemArchitecture(arch string) SystemArchitecture {
 		return Arm64
 	case string(Amd64):
 		return Amd64
+	case "":
```

Comment:
Shouldn't we just change default: to return DefaultArch? It seems surprising to differentiate.

Reply:
I decided to go with using DefaultArch as default:, given that incorrect value in this place will lead to cluster autoscaler crash in main loop.